crossdock/crossdock_test.go

```diff
@@ -21,33 +21,22 @@
 package main
 
 import (
-	"encoding/json"
-	"log"
 	"net/url"
 	"testing"
-	"time"
 
+	"github.com/yarpc/yarpc-go/crossdock-go/crossdock"
```

Comment:
This will end up being one of the following:

```
import "github.com/crossdock/crossdock-go/crossdock"
import "go.uber.com/crossdock"
import "go.yarpc.io/crossdock"
```

Obv the canonical import paths are more desirable :)